tapdance/logger.go

```diff
@@ -17,13 +17,16 @@ func (f *formatter) Format(entry *logrus.Entry) ([]byte, error) {
 var logrusLogger *logrus.Logger
 var initLoggerOnce sync.Once
 
+const build string = "[BUILD]"
+
 // Logger is an access point for TapDance-wide logger
 func Logger() *logrus.Logger {
 	initLoggerOnce.Do(func() {
 		logrusLogger = logrus.New()
 		logrusLogger.Formatter = new(formatter)
 		logrusLogger.Level = logrus.InfoLevel
 		//logrusLogger.Level = logrus.DebugLevel
+		logrusLogger.Infof("Running gotapdance build %s", build)
```

Comment:
What's the purpose of this?
This line will always be printed when `Logger()` is first called, even if tapdance was built a long time ago, and it's the Android App trying to log things.

Reply:
Yes this is intended. So we know which gotapdance build the Psiphon client is running

Reply:
Oh, I see the 6a8b2cf5aaefeaece4729c45db800969c865481f
This is a great idea!